src/modules/NtpClient.rb

```diff
@@ -1045,42 +1045,53 @@ def Summary
       summary
     end
 
+
     # Test if specified NTP server answers
     # @param [String] server string host name or IP address of the NTP server
+    # @return [Boolean] true if NTP server answers properly
+    def reachable_ntp_server?(server)
+      # testing the server using IPv4 and then using IPv6 protocol
+      # bug #74076, Firewall could have been blocked IPv6
+      return true if sntp_test(server) == 0
+
+      sntp_test(server, 6) == 0
+    end
+
+    # -K /dev/null: use /dev/null as KoD history file (if not specified,
```

Comment:
These comments about sntp syntax are probably better placed before the call, rather than in the method description. They are implementation details of the test.